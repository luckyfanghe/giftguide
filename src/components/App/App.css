@font-face {
  font-family: GothamRndSSm-Bold;
  src: url("../../assets/fonts/gotham-cufonfonts/GothamRndSSm-Bold.otf");
}
@font-face {
  font-family: GothamRndSSm-BoldItalic;
  src: url("../../assets/fonts/gotham-cufonfonts/GothamRndSSm-BoldItalic.otf");
}
@font-face {
  font-family: GothamRndSSm-Book;
  src: url("../../assets/fonts/gotham-cufonfonts/GothamRndSSm-Book.otf");
}
@font-face {
  font-family: GothamRndSSm-BookItalic;
  src: url("../../assets/fonts/gotham-cufonfonts/GothamRndSSm-BookItalic.otf");
}
@font-face {
  font-family: GothamRndSSm-Light;
  src: url("../../assets/fonts/gotham-cufonfonts/GothamRndSSm-Light.otf");
}
@font-face {
  font-family: GothamRndSSm-LightItalic;
  src: url("../../assets/fonts/gotham-cufonfonts/GothamRndSSm-LightItalic.otf");
}
@font-face {
  font-family: GothamRndSSm-Medium;
  src: url("../../assets/fonts/gotham-cufonfonts/GothamRndSSm-Medium.otf");
}
@font-face {
  font-family: GothamRndSSm-MediumItalic;
  src: url("../../assets/fonts/gotham-cufonfonts/GothamRndSSm-MediumItalic.otf");
}

body {
  background-color: rgba(25, 34, 48, 0.05);
  min-height: 1100px;
  margin: 0;
  padding: 0;
  font-family: "GothamRndSSm-Medium";
  font-style: normal;
  font-weight: 400;
}
