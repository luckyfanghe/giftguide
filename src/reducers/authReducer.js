// @flow
import types from "../actions/types";

const initialState = {
  sidebarOpened: true,
  sidebarStyle: "login",
  authState: ""
};
function authReducer(state = initialState, action) {
  switch (action.type) {
    case types.AUTH.OPENSIDEBAR:
      return {
        ...state,
        sidebarOpened: true,
        sidebarStyle: action.sidebarStyle
      };
    case types.AUTH.CLOSESIDEBAR:
      return {
        ...state,
        sidebarOpened: false,
        sidebarStyle: action.sidebarStyle
      };
    case types.AUTH.DOINGAUTH:
      return {
        ...state,
        authState: action.state
      };
    default:
      return {
        ...state
      };
  }
}

export default authReducer;
