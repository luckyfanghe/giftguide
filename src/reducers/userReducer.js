// @flow
import types from "../actions/types";

const initialState = {
  user: null,
  userState: "",
  userStateDes: ""
};
function userReducer(state = initialState, action) {
  switch (action.type) {
    case types.USER.UPDATE:
      return {
        ...state,
        user: action.userInfo
      };
    case types.USER.LOGINFAILED:
      return {
        ...state,
        userState: "login failed",
        userStateDes: action.state
      };
    case types.USER.LOGINSUCCESS:
      return {
        ...state,
        userState: "login success",
        userStateDes: ""
      };
    case types.USER.SIGNUPFAILED:
      return {
        ...state,
        userState: "signup failed",
        userStateDes: action.state
      };
    case types.USER.SIGNUPSUCCESS:
      return {
        ...state,
        userState: "signup success",
        userStateDes: ""
      };
    default:
      return {
        ...state
      };
  }
}

export default userReducer;
