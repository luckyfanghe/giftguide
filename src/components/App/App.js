import React, { Component, lazy, Suspense } from "react";
import { Route, Redirect, Switch, BrowserRouter } from "react-router-dom";
import { connect, Provider } from "react-redux";
import { bindActionCreators } from "redux";
import Loadable from "react-loadable";

// import { history } from "./../../stores";
import * as routes from "../../constants/routes";
import "./App.css";
import "../Styles/font.css";

import PageFactory from "../Pages/PageFactory";

import DashboardPage from "../Pages/DashboardPage";
import AboutPage from "../Pages/AboutPage";
import AccountPage from "../Pages/AccountPage";
import AccountSettingPage from "../Pages/AccountSettingPage";
import UpdateWishItemPage from "../Pages/UpdateWishItemPage";
import AddWishItemPage from "../Pages/AddWishItemPage";
import BlogPage from "../Pages/BlogPage";
import CalendarPage from "../Pages/CalendarPage";
import FeedbackPage from "../Pages/FeedbackPage";
import FriendsPage from "../Pages/FriendsPage";
import GiftGuidesPage from "../Pages/GiftGuidesPage";
import PrivacyPolicyPage from "../Pages/PrivacyPolicyPage";
import ProfilePage from "../Pages/ProfilePage";
import SearchPage from "../Pages/SearchPage";
import WishListPage from "../Pages/WishListPage";
import ContactPage from "../Pages/ContactPage";
import delay from "delay";
const loading = () => <div>Loading</div>;

class App extends Component {
  constructor(props) {
    super(props);
  }

  componentWillMount() {}

  render() {
    const DashboardPageAugmented = () => (
      <PageFactory>
        <DashboardPage />
      </PageFactory>
    );
    const AboutPageAugmented = () => (
      <PageFactory>
        <AboutPage />
      </PageFactory>
    );
    const AccountPageAugmented = () => (
      <PageFactory>
        <AccountPage />
      </PageFactory>
    );
    const AccountSettingPageAugmented = () => (
      <PageFactory>
        <AccountSettingPage />
      </PageFactory>
    );
    const UpdateWishItemPageAugmented = () => (
      <PageFactory>
        <UpdateWishItemPage />
      </PageFactory>
    );
    const BlogPageAugmented = () => (
      <PageFactory>
        <BlogPage />
      </PageFactory>
    );
    const CalendarPageAugmented = () => (
      <PageFactory>
        <CalendarPage />
      </PageFactory>
    );
    const FeedbackPageAugmented = () => (
      <PageFactory>
        <FeedbackPage />
      </PageFactory>
    );
    const FriendsPageAugmented = () => (
      <PageFactory>
        <FriendsPage />
      </PageFactory>
    );
    const WishListPageAugmented = () => (
      <PageFactory>
        <WishListPage />
      </PageFactory>
    );
    const SearchPageAugmented = () => (
      <PageFactory>
        <SearchPage />
      </PageFactory>
    );
    const ProfilePageAugmented = () => (
      <PageFactory>
        <ProfilePage />
      </PageFactory>
    );
    const GiftGuidesPageAugmented = () => (
      <PageFactory>
        <GiftGuidesPage />
      </PageFactory>
    );
    const PrivacyPolicyPageAugmented = () => (
      <PageFactory>
        <PrivacyPolicyPage />
      </PageFactory>
    );
    const ContactPageAugmented = () => (
      <PageFactory>
        <ContactPage />
      </PageFactory>
    );
    const AddWishItemPageAugmented = () => (
      <PageFactory>
        <AddWishItemPage />
      </PageFactory>
    );
    // prettier-ignore
    return (
        <BrowserRouter>
          <React.Fragment>
            <div className="app"  data-testid="App">
              <Switch>
                <Redirect exact from="/" to={routes.DASHBOARD_PAGE} />
                <Route exact path={routes.DASHBOARD_PAGE} component={DashboardPageAugmented} />
              </Switch>
              <Route exact path={ routes.SEARCH_PAGE } component={ SearchPageAugmented } />
              <Route exact path={ routes.WISH_LIST_PAGE } component={ WishListPageAugmented } />
              <Route exact path={routes.ABOUT_PAGE} component={AboutPageAugmented} />
              <Route exact path={routes.ACCOUNT_PAGE} component={AccountPageAugmented} />
              <Route exact path={routes.ACCOUNT_SETTING_PAGE}	component={AccountSettingPageAugmented} />
              <Route exact path={routes.UPDATE_WISH_ITEM_PAGE}	component={UpdateWishItemPageAugmented} />
              <Route exact path={routes.BLOG_PAGE}	component={BlogPageAugmented} />
              <Route exact path={routes.CALENDAR_PAGE} component={CalendarPageAugmented} />
              <Route exact path={routes.FEEDBACK_PAGE}	component={FeedbackPageAugmented} />
              <Route exact path={routes.FRIENDS_PAGE} component={ FriendsPageAugmented } />
              <Route exact path={routes.PROFILE_PAGE}	component={ProfilePageAugmented} />
              <Route exact path={routes.GIFTGUIDES_PAGE}	component={GiftGuidesPageAugmented} />
              <Route exact path={routes.PRIVACY_POLICY_PAGE}	component={PrivacyPolicyPageAugmented} />
              <Route exact path={routes.CONTACT_PAGE}	component={ContactPageAugmented} />
              <Route exact path={routes.ADD_WISH_ITEM_PAGE}	component={AddWishItemPageAugmented} />
            </div>
          </React.Fragment>
        </BrowserRouter>
    );
  }
}

const mapStateToProps = state => {
  return {
    user: state.userReducer.user,
    userState: state.userReducer.userState
  };
};

// const mapDispatchToProps = dispatch =>
//   bindActionCreators({ userRequest }, dispatch);
const mapDispatchToProps = dispatch => bindActionCreators({}, dispatch);

// export default App;
export default connect(mapStateToProps, mapDispatchToProps)(App);
