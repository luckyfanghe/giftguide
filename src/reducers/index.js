import { combineReducers } from "redux";
import { connectRouter } from "connected-react-router";

import userReducer from "./userReducer";
import authReducer from "./authReducer";

export default combineReducers({
  userReducer,
  authReducer
});
