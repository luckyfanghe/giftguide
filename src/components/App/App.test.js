import React from "react";
import { render } from "@testing-library/react";
import renderer from "react-test-renderer";
import App from "./App";

import { Provider } from "react-redux";
import { composeWithDevTools } from "redux-devtools-extension";
import { createStore, applyMiddleware, compose } from "redux";
import createSagaMiddleware from "redux-saga";
import reducers from "../../reducers";
import rootSaga from "../../sagas";

const sagaMiddleware = createSagaMiddleware();
const enhancers = composeWithDevTools(applyMiddleware(sagaMiddleware));

const store = createStore(reducers, enhancers);

sagaMiddleware.run(rootSaga);

test("renders app", () => {
  const { queryByTestId } = render(
    <Provider store={store}>
      <App />
    </Provider>
  );
  expect(queryByTestId("App")).toBeTruthy();
});

describe("App", () => {
  test("App snapshot renders ", () => {
    const component = renderer.create(
      <Provider store={store}>
        <App />
      </Provider>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
