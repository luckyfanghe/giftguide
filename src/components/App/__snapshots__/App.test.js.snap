// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App snapshot renders  1`] = `
<div
  className="app"
  data-testid="App"
>
  <div
    className="body"
    id="outer-container"
  >
    <div>
      <div>
        <div
          className="bm-overlay"
          onClick={[Function]}
          style={
            Object {
              "MozTransform": "",
              "MsTransform": "",
              "OTransform": "",
              "WebkitTransform": "",
              "background": "rgba(0, 0, 0, 0.3)",
              "height": "100%",
              "opacity": 1,
              "position": "fixed",
              "transform": "",
              "transition": "opacity 0.3s",
              "width": "100%",
              "zIndex": 1000,
            }
          }
        />
        <div
          className="bm-menu-wrap"
          id="slide"
          style={
            Object {
              "MozTransform": "",
              "MsTransform": "",
              "OTransform": "",
              "WebkitTransform": "",
              "height": "100%",
              "position": "fixed",
              "right": 0,
              "transform": "",
              "transition": "all 0.5s",
              "width": "30%",
              "zIndex": 1100,
            }
          }
        >
          <div
            className="bm-menu"
            style={
              Object {
                "boxSizing": "border-box",
                "height": "100%",
                "overflow": "auto",
              }
            }
          >
            <nav
              className="bm-item-list"
              style={
                Object {
                  "height": "100%",
                }
              }
            >
              <div
                className="sc-fzXfMz llUWBW"
                data-testid="SlideBar"
              >
                <img
                  alt="logo"
                  className="sc-fzXfMB eezMiq"
                  src="Logo-large.webp"
                />
                <div
                  className="sc-fzXfMC llZPlh"
                >
                  WELCOME TO GIFT GIUDE
                </div>
                <form
                  className=""
                  noValidate={true}
                  onSubmit={[Function]}
                >
                  <div
                    className="form-group"
                  >
                    <label
                      className="form-label"
                      htmlFor="formBasicEmail"
                    >
                      Email
                    </label>
                    <input
                      className="form-control"
                      id="formBasicEmail"
                      onClick={[Function]}
                      required={true}
                      type="email"
                    />
                  </div>
                  <div
                    className="form-group"
                  >
                    <label
                      className="form-label"
                      htmlFor="formBasicPassword"
                    >
                      Password
                    </label>
                    <input
                      className="form-control"
                      id="formBasicPassword"
                      onClick={[Function]}
                      required={true}
                      type="password"
                    />
                  </div>
                  <button
                    className="auth-button btn btn-primary"
                    disabled={false}
                    type="submit"
                  >
                    LOG IN
                  </button>
                </form>
                <div
                  className="sc-fzXfNc iUzckS"
                >
                  
                </div>
                <div
                  className="sc-fzXfME gzQAqq"
                >
                   
                  test
                </div>
                <div
                  className="sc-fzXfME gzQAqq"
                  onClick={[Function]}
                >
                  create an account
                </div>
                <div
                  className="sc-fzXfME gzQAqq"
                >
                  login with google or facebook
                </div>
                <div
                  className="sc-fzXfNd igeocn"
                >
                  
                </div>
              </div>
            </nav>
          </div>
          <div
            style={Object {}}
          >
            <div
              className="bm-cross-button"
              style={
                Object {
                  "height": 24,
                  "position": "absolute",
                  "right": 8,
                  "top": 8,
                  "width": 24,
                }
              }
            >
              <span
                style={
                  Object {
                    "position": "absolute",
                    "right": "14px",
                    "top": "6px",
                  }
                }
              >
                <span
                  className="bm-cross"
                  style={
                    Object {
                      "height": 14,
                      "position": "absolute",
                      "transform": "rotate(45deg)",
                      "width": 3,
                    }
                  }
                />
                <span
                  className="bm-cross"
                  style={
                    Object {
                      "height": 14,
                      "position": "absolute",
                      "transform": "rotate(-45deg)",
                      "width": 3,
                    }
                  }
                />
              </span>
              <button
                onClick={[Function]}
                style={
                  Object {
                    "background": "transparent",
                    "border": "none",
                    "cursor": "pointer",
                    "fontSize": 0,
                    "height": "100%",
                    "left": 0,
                    "margin": 0,
                    "padding": 0,
                    "position": "absolute",
                    "top": 0,
                    "width": "100%",
                  }
                }
                tabIndex={0}
              >
                Close Menu
              </button>
            </div>
          </div>
        </div>
        <div
          style={Object {}}
        >
          <div
            className="bm-burger-button"
            style={
              Object {
                "zIndex": 1000,
              }
            }
          >
            <span>
              <span
                className="bm-burger-bars"
                style={
                  Object {
                    "height": "20%",
                    "left": 0,
                    "opacity": 1,
                    "position": "absolute",
                    "right": 0,
                    "top": "0%",
                  }
                }
              />
              <span
                className="bm-burger-bars"
                style={
                  Object {
                    "height": "20%",
                    "left": 0,
                    "opacity": 1,
                    "position": "absolute",
                    "right": 0,
                    "top": "40%",
                  }
                }
              />
              <span
                className="bm-burger-bars"
                style={
                  Object {
                    "height": "20%",
                    "left": 0,
                    "opacity": 1,
                    "position": "absolute",
                    "right": 0,
                    "top": "80%",
                  }
                }
              />
            </span>
            <button
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
              style={
                Object {
                  "background": "transparent",
                  "border": "none",
                  "cursor": "pointer",
                  "fontSize": 0,
                  "height": "100%",
                  "left": 0,
                  "margin": 0,
                  "padding": 0,
                  "position": "absolute",
                  "top": 0,
                  "width": "100%",
                }
              }
            >
              Open Menu
            </button>
          </div>
        </div>
      </div>
    </div>
    <main
      id="page-wrap"
    >
      <div
        className="sc-AykKE XlglN"
        data-testid="Header"
      >
        <div
          className="sc-AykKF jSinek"
        >
          <div
            className="sc-AykKG iQLzuy"
          >
            <img
              alt="logo"
              src="Logo.webp"
            />
            <span
              className="sc-AykKH kvjmHj"
            >
              THE GIFT GUIDE
            </span>
          </div>
          <div
            className="sc-AykKI cYsRws"
          >
            <div
              className="sc-AykKJ bvGAFD"
            >
              <input
                className="form-control"
                onKeyDown={[Function]}
                placeholder="Search"
                type="text"
              />
            </div>
            <img
              alt="search"
              onClick={[Function]}
              src="search-icon.webp"
            />
            <span
              className="sc-AykKK evIedy"
            >
              +
            </span>
            <img
              alt="avatar"
              className="sc-fzXfLO gnWvlK"
              src="avatar.webp"
            />
          </div>
        </div>
        <div
          className="nab-bar"
        >
          <nav
            className="nab-bar-bar navbar navbar-expand-lg navbar-light bg-light"
          >
            <button
              aria-controls="basic-navbar-nav"
              aria-label="Toggle navigation"
              className="navbar-toggler collapsed"
              onClick={[Function]}
              type="button"
            >
              <span
                className="navbar-toggler-icon"
              />
            </button>
            <div
              aria-expanded={null}
              className="nab-bar-link navbar-collapse collapse"
              id="basic-navbar-nav"
            >
              <div
                className="mr-auto navbar-nav"
                onKeyDown={[Function]}
              >
                <a
                  className="nab-link nav-link"
                  data-rb-event-key={null}
                  disabled={false}
                  href="/dashboard"
                  onClick={[Function]}
                >
                  DASHBOARD
                </a>
                <a
                  className="nab-link nav-link"
                  data-rb-event-key={null}
                  disabled={false}
                  href="/wish_list"
                  onClick={[Function]}
                >
                  WISH LIST
                </a>
                <a
                  className="nab-link nav-link"
                  data-rb-event-key={null}
                  disabled={false}
                  href="/friends"
                  onClick={[Function]}
                >
                  FRIENDS
                </a>
                <a
                  className="nab-link nav-link"
                  data-rb-event-key={null}
                  disabled={false}
                  href="/calendar"
                  onClick={[Function]}
                >
                  CALENDAR
                </a>
                <a
                  className="nab-link nav-link"
                  data-rb-event-key={null}
                  disabled={false}
                  href="/gift_guides"
                  onClick={[Function]}
                >
                  GIFT GUIDES
                </a>
                <a
                  className="nab-link nav-link"
                  data-rb-event-key={null}
                  disabled={false}
                  href="/blog"
                  onClick={[Function]}
                >
                  BLOG
                </a>
                <a
                  className="nab-link nav-link"
                  data-rb-event-key={null}
                  disabled={false}
                  href="/contact"
                  onClick={[Function]}
                >
                  CONTACT
                </a>
                <a
                  className="nab-link nav-link"
                  data-rb-event-key={null}
                  disabled={false}
                  href="/about"
                  onClick={[Function]}
                >
                  ABOUT
                </a>
              </div>
            </div>
          </nav>
        </div>
      </div>
      <div
        className="sc-fzXfNe bRYtIL"
      >
        <div
          className="sc-fzXfNK gQxaxW"
          data-testid="Dashboard"
        >
          <div
            className="sc-fzXfNL gxhxA"
          >
            <div>
              SIMPLIFY GIFTING AND ENSURE DELIGHTED
            </div>
            <div>
              FACES ON SPECIAL OCCASIONS
            </div>
          </div>
          <div
            className="sc-fzXfNM cSeHjg"
          >
            <div
              className="sc-fzXfNf ftMzrU"
              data-testid="Card"
              id="card-1"
            >
              <div
                className="sc-fzXfNg enCQAw"
              >
                CALENDAR
              </div>
              <div
                className="sc-fzXfNi ihsoGO"
              >
                <div
                  className="sc-fzXfNj hRCHJC"
                  id="des-1"
                >
                  gift guide creates a customized
                </div>
                <div
                  className="sc-fzXfNj hRCHJC"
                  id="des-2"
                >
                  calendar for you with all your loved
                </div>
                <div
                  className="sc-fzXfNj hRCHJC"
                  id="des-3"
                >
                  one's most important dates, in one
                </div>
                <div
                  className="sc-fzXfNj hRCHJC"
                  id="des-4"
                >
                  easy location
                </div>
              </div>
              <button
                className="sc-fzXfNh iBOVKk"
                type="button"
              >
                 
                ADD FRIENDS
                 
              </button>
            </div>
            <div
              className="sc-fzXfNf ftMzrU"
              data-testid="Card"
              id="card-2"
            >
              <div
                className="sc-fzXfNg enCQAw"
              >
                WISH LIST
              </div>
              <div
                className="sc-fzXfNi ihsoGO"
              >
                <div
                  className="sc-fzXfNj hRCHJC"
                  id="des-5"
                >
                  gift guide users set up profiles
                </div>
                <div
                  className="sc-fzXfNj hRCHJC"
                  id="des-6"
                >
                  containing their favorite stores, wish
                </div>
                <div
                  className="sc-fzXfNj hRCHJC"
                  id="des-7"
                >
                  lists and items they love, so you no
                </div>
                <div
                  className="sc-fzXfNj hRCHJC"
                  id="des-8"
                >
                  longer have to ask for ideas or sizes
                </div>
              </div>
              <button
                className="sc-fzXfNh iBOVKk"
                type="button"
              >
                 
                ADD TO WISH LIST
                 
              </button>
            </div>
            <div
              className="sc-fzXfNf ftMzrU"
              data-testid="Card"
              id="card-3"
            >
              <div
                className="sc-fzXfNg enCQAw"
              >
                REMINDERS
              </div>
              <div
                className="sc-fzXfNi ihsoGO"
              >
                <div
                  className="sc-fzXfNj hRCHJC"
                  id="des-9"
                >
                  gift guide sends you reminders before
                </div>
                <div
                  className="sc-fzXfNj hRCHJC"
                  id="des-10"
                >
                  your loved ones' events with their
                </div>
                <div
                  className="sc-fzXfNj hRCHJC"
                  id="des-11"
                >
                  wish lists and favorite stores so you
                </div>
                <div
                  className="sc-fzXfNj hRCHJC"
                  id="des-12"
                >
                  never miss an important date again
                </div>
              </div>
              <button
                className="sc-fzXfNh iBOVKk"
                type="button"
              >
                 
                SET UP REMINDERS
                 
              </button>
            </div>
          </div>
          <div
            className="sc-fzXfNN dlhDcb"
          >
            <div
              className="sc-fzXfNk bmKW"
              data-testid="Banner"
            >
              <img
                alt="banner"
                className="sc-fzXfNl ccAnUB"
                src="banner2.webp"
              />
              <button
                className="sc-fzXfNJ biUSay"
                type="button"
              >
                GET STARTED
              </button>
            </div>
            <div
              className="sc-fzXfNk bmKW"
              data-testid="Banner"
            >
              <img
                alt="banner"
                className="sc-fzXfNl ccAnUB"
                src="banner1.webp"
              />
              <button
                className="sc-fzXfNJ biUSay"
                type="button"
              >
                BROWSE GIFT IDEAS
              </button>
            </div>
          </div>
        </div>
      </div>
      <footer>
        <div
          className="sc-fzXfLP bIeXHY"
          data-testid="Footer"
        >
          <div
            className="sc-fzXfLQ dKSwrA"
          >
            <div
              className="sc-fzXfLR ekUgGh"
            >
              CONNECT WITH GIFT GUIDE
            </div>
            <div
              className="sc-fzXfLS kNHjih"
            >
              <div
                className="sc-fzXfLT cdQoeC"
              >
                <img
                  src="https://static.wixstatic.com/media/8f6f59264a094af0b46e9f6c77dff83e.png/v1/fill/w_35,h_35,al_c,q_85,usm_0.66_1.00_0.01/8f6f59264a094af0b46e9f6c77dff83e.webp"
                />
              </div>
              <div
                className="sc-fzXfLT cdQoeC"
              >
                <img
                  src="https://static.wixstatic.com/media/01c3aff52f2a4dffa526d7a9843d46ea.png/v1/fill/w_35,h_35,al_c,q_85,usm_0.66_1.00_0.01/01c3aff52f2a4dffa526d7a9843d46ea.webp"
                />
              </div>
              <div
                className="sc-fzXfLT cdQoeC"
              >
                <img
                  src="https://static.wixstatic.com/media/0fdef751204647a3bbd7eaa2827ed4f9.png/v1/fill/w_35,h_35,al_c,q_85,usm_0.66_1.00_0.01/0fdef751204647a3bbd7eaa2827ed4f9.webp"
                />
              </div>
              <div
                className="sc-fzXfLT cdQoeC"
              >
                <img
                  src="https://static.wixstatic.com/media/c7d035ba85f6486680c2facedecdcf4d.png/v1/fill/w_35,h_35,al_c,q_85,usm_0.66_1.00_0.01/c7d035ba85f6486680c2facedecdcf4d.webp"
                />
              </div>
            </div>
            <div
              className="sc-fzXfLU bcnaoV"
            >
              <div
                className="sc-fzXfLV lba-DNo"
              >
                CONTACT US
              </div>
              <div
                className="sc-fzXfLV lba-DNo"
              >
                TEAMS OF USE / PRIVACY
              </div>
              <div
                className="sc-fzXfLV lba-DNo"
              >
                FEEDBACK
              </div>
            </div>
          </div>
          <div
            className="sc-fzXfLW fXVMqY"
          >
            <div
              className="sc-fzXfLR ekUgGh"
            >
              INVITE FRIENDS TO GIFT GUIDE
            </div>
            <div
              className="sc-fzXfLX csdvWa"
            >
              enter your loved one's email to invite
            </div>
            <div
              className="sc-fzXfMv MVnpV"
            >
              <div
                className="sc-fzXfMw kQchwh"
              >
                <input
                  className="form-control"
                  type="text"
                />
              </div>
              <button
                className="sc-fzXfMx bQMxPD"
                type="button"
              >
                INVITE
              </button>
            </div>
            <div
              className="sc-fzXfMy bGETPR"
            >
              2019 TM GIFT GUIDE LLC
            </div>
          </div>
        </div>
      </footer>
    </main>
  </div>
</div>
`;
